crates/solidity/inputs/language/src/definition.rs

```diff
@@ -4,7 +4,7 @@ codegen_language_macros::compile!(Language(
     name = Solidity,
     documentation_dir = "crates/solidity/inputs/language/docs",
     binding_rules_file = "crates/solidity/inputs/language/bindings/rules.msgb",
-    file_extension = ".sol",
+    file_extension = ".sol", // TODO: This should be moved to the Solidity-specific 'extensions' sub-module.
```

Comment:
I was thinking it may make sense to put this in the new `LanguageFacts` struct. WDYT?

Reply:
mmm, What is the use case to expose this in the public API?